File: alice/paskills/my_alice/src/main/java/ru/yandex/alice/paskills/my_alice/blocks/recommender/RecommenderResponse.java
package ru.yandex.alice.paskills.my_alice.blocks.recommender;

import java.util.List;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;
import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;

@Data
public class RecommenderResponse {

    private static final Logger logger = LogManager.getLogger();

    private final List<RecommenderItem> items;
    @JsonProperty("recommendation_type")
    private final String recommendationType;
    @JsonProperty("recommendation_source")
    private final String recommendationSource;


}

File: alice/paskills/dialogovo/common/src/main/java/service/recommender/RecommenderResponse.kt
package ru.yandex.alice.paskill.dialogovo.service.recommender

import com.fasterxml.jackson.annotation.JsonProperty

data class RecommenderResponse(
    @JsonProperty("recommendation_type") val recommendationType: String,
    @JsonProperty("recommendation_source") val recommendationSource: String,
    val items: List<RecommenderItem> = emptyList(),
)
